galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/RunResult.java

```diff
@@ -0,0 +1,75 @@
+package dev.galasa.api.run;
+
+import java.time.Instant;
+
+
+public class RunResult {
+	
+	private final String runName;
+	private final String testName;
+	private final String testShortName;
+	private final String bundle;
+	private final String requestor;
+	private final String result;
+	private final String status;
+	private final Instant queued;
+	private final Instant start;
+	private final Instant end;
+	
+	public RunResult(String runName, String testName, String testShortName, String bundle,
```

Comment:
You will need the appropriate set* so the class can be used at the other end

Reply:
So should I remove the parameters from the constructor and just have set methods?

Reply:
either,  up to you really,  you need the set* for json to work properly.   but you want to maintain the constructor as well.....